src/targets/gpu/pack_int8_args.cpp

```diff
@@ -177,8 +178,13 @@ void pack_int8_args::apply(module& m) const
                 m.replace_instruction(ins, ins->get_operator(), inputs);
             }
         }
-        else if(ins->name() == "gpu::quant_convolution")
+        else if(ins->name() == "gpu::miopen_convolution")
```

Comment:
Templating the class will make it simpler to check the operator name.

Reply:
Done.